lib/src/components/NxTile/types.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { HTMLAttributes, ReactNode } from 'react';
+
+export interface NxTileHeaderTitleProps extends HTMLAttributes<HTMLDivElement> {
+  children: ReactNode;
```

Comment:
Is this not already in `HTMLAttributes<HTMLDivElement>`?

Reply:
> Is this not already in `HTMLAttributes<HTMLDivElement>`?

Yup! I fixed that as part of: ebd40de14d6f870840366f6a992f2ed1f0bb41b
(Also updated the ones in NxDrawer)